templates/types/extractor/fastapi/app/ui/components/schema_editor.py

```diff
@@ -9,7 +9,7 @@ def handle_schema(self, schema: str):
         self.schema = schema
 
     def init_schema(self):
-        self.schema = SchemaService().get_current_schema()
+        self.schema = Output.schema_json(indent=2)
```

Comment:
Output won't exist later, but the service can provide a default schema
```suggestion
        self.schema = ExtractorService.default_schema()
```